src/shims/unix/fs.rs

```diff
@@ -1086,8 +1088,18 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                 let name_bytes = name.as_encoded_bytes();
                 let name_len = u64::try_from(name_bytes.len()).unwrap();
```

Comment:
There's a comment above here, documenting the struct layout, that is now outdated.